### PR TITLE
Create Admin: dashboard, users

### PR DESCRIPTION
**Pull request by nguyenduyta**

### CODE REVIEW

File: app/views/layouts/admin/_shim.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+<script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
```
Comment (dieunb) [inline app/views/layouts/admin/_shim.html.erb]: indent

File: app/views/layouts/admin/_shim.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+</script>
```
Comment (dieunb) [inline app/views/layouts/admin/_shim.html.erb]: indent

**Comment by dieunb** [comment]
OK

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -2,4 +2,10 @@ module SessionsHelper
   def current_user? user
     user == current_user
   end
+
+  def logged_in_admin?
+    unless current_user.admin?
+      redirect_to admin_authenticate_path
```
Comment (chuanhtuan) [inline app/helpers/sessions_helper.rb]: anh nghĩ render thì hay hơn redirect

File: config/routes.rb [hunk 4]
```diff
@@ -11,6 +11,11 @@
     resources :lessons, only: [:new, :create, :show]
   end
   resources :words, only: :index
+  get 'admin_authenticate/' => 'static_pages#admin_authenticate'
```
Comment (chuanhtuan) [inline config/routes.rb]: remove

**Pull request merged**